.github/workflows/pull_request.yml

```diff
@@ -64,9 +64,14 @@ jobs:
           path: ~/.m2
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
+      - name: Install firefox and set FIREFOX_FOLDER env variable
+        run: |
+          wget https://ftp.mozilla.org/pub/firefox/releases/60.9.0esr/linux-x86_64/en-US/firefox-60.9.0esr.tar.bz2
+          tar -xf firefox-60.9.0esr.tar.bz2
+          echo "FIREFOX_FOLDER=`pwd`/firefox" >> $GITHUB_ENV
```

Comment:
Is this correct?

Reply:
I mean setting the FIREFOX_FOLDER var.

Reply:
I think it is correct. `$FIREFOX_FOLDER` is not directly used but `${{ env.FIREFOX_FOLDER }}` is used. And according to this https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable looks good.